source/backend/api/Services/TakeService.cs

```diff
@@ -112,15 +110,15 @@ public IEnumerable<PimsTake> UpdateAcquisitionPropertyTakes(long acquisitionFile
             var currentProperty = _acqFileRepository.GetProperty(acquisitionFilePropertyId);
             var currentTakes = _takeRepository.GetAllByPropertyId(currentProperty.PropertyId);
 
-            var completedTakes = currentTakes.Where(t => t.TakeStatusTypeCode == "COMPLETE").ToList();
+            var completedTakes = currentTakes.Union(takes)
```

Comment:
I suppose this works for now - I think this will become problematic when we add completed take deletion.